src/System.Globalization.Calendars/tests/CalendarHelpers.cs

```diff
@@ -161,49 +164,76 @@ public static IEnumerable<object[]> DateTime_TestData()
         }
 
         [Theory]
-        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year)]
+        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year, false)]
         public static void GetDaysInYear_Invalid(Calendar calendar, int year, int month, int day, int era, string exceptionParamName)
         {
             Assert.Throws<ArgumentOutOfRangeException>(exceptionParamName, () => calendar.GetDaysInYear(year, era));
         }
 
         [Theory]
-        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year)]
+        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year, false)]
```

Comment:
I can just rename the name of the parameter if this making any confusion. I can call it "ignoreJapaneseLunisolarCalendar" for instance. does this clear the confusion? 

---
In reply to: [95621267](https://github.com/dotnet/corefx/pull/15056#discussion_r95621267) [](ancestors = 95621267)